src/casp/cromwell/README.md

```diff
@@ -0,0 +1,87 @@
+# CAS Cromwell Directory
```

Comment:
Please rename `casp/cromwell` to `/casp/wdl_scripts`.  Cromwell is the executioner, WDL is the scripting language.